docs/utilities/jmespath_functions.md

```diff
@@ -0,0 +1,166 @@
+---
+title: JMESPath Functions
+description: Utility
+---
+
+You might have events or responses that contain non-encoded JSON, where you need to decode so that you can access portions of the object or ensure the Powertools utility recieves a JSON object
```

Comment:
Want to add a connection to the validation and idempotency utils now that this util is separated out, as that is the primary use case for it.
```suggestion
You might have events or responses that contain non-encoded JSON, where you need to decode so that you can access portions of the object or ensure the Powertools utility receives a JSON object. This is a common use case when using the [validation](/utilities/validation) or [idempotency](/utilities/idempotency) utilities.  
```